README.md

```diff
@@ -28,3 +28,11 @@ node js/bin/metricly.js
 npm install -g ts-node
 ts-node ts/bin/metricly.ts
 ```
+
+## Testing
+// run tests
+```npm test```
```

Comment:
Should be `yarn test`. There isn't a functional difference, just keeping in line with using Yarn in this project.